docs/source/conf.py

```diff
@@ -177,7 +177,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['images']
```

Comment:
if we're going to use the static folder to store images, let's move the contents of the `source/images` folder into `_static/images` so we're consistent